TYPO3.Flow/Classes/TYPO3/Flow/Package/PackageInterface.php

```diff
@@ -152,7 +154,7 @@ public function getDocumentationPath();
      * Returns the available documentations for this package
      *
      * @return array Array of \TYPO3\Flow\Package\Documentation
-     * @api
+     * @deprecated To be removed in Flow 4.0
```

Comment:
For this and all above (except getNamespace): what are the replacements to get the same information? Since those functions where @api there might be code around that depends on retrieving that information in some way.